README.md

```diff
@@ -51,6 +53,19 @@ After you successfully set up the project, you will have to go through the follo
    * use it as the return value of the method `getBotToken()` inside `StickerifyBot`
 3. Run the `Main` class to start the bot, it will be now able to answer messages in Telegram
 
+## Launch The Bot Using Docker
+
+1. Install [Docker](https://docs.docker.com/get-docker/).
+2. Run Docker,passing it the argument variable with the following command:
+   ```shell
+   docker build -t build-name --build-arg name=arg-name 
+   ```
+3. In our case, the specific command will be:
+   ```shell
+   docker build -t stickerify --build-arg STICKERIFY_TOKEN={{TOKEN}} 
+   docker run stickerify
+   ``` where {{TOKEN}} is the token which will be retrieved from BotFather.
```

Comment:
I would put the "where" etc on a new line. Also, BotFather should be linked: `[BotFather](https://t.me/BotFather)`